src/core/connect-extension/conn-ext-trim-state.ts

```diff
@@ -106,7 +106,7 @@ const trimPreferences = (preferences: IPrefState): IExtStorage.IStoragePreferenc
         currency: preferences.currency,
         testNet: preferences.testNet,
         networks: cloneDeep(preferences.networks),
-        blockchains: Object.keys(preferences.blockchains)
+        blockchains: cloneDeep(preferences.blockchains)
```

Comment:
Since we change the type here, shouldn't we do changes on extension side, where we load this data in state.
Also is this change backward compatible ?